src/services/lnd/schema.ts

```diff
@@ -0,0 +1,58 @@
+import * as mongoose from "mongoose"
+
+const Schema = mongoose.Schema
+
+const confirmedDetailsSchema = new Schema<LnPaymentConfirmedDetails>({
+  confirmedAt: {
+    type: Date,
+    required: true,
+  },
+  destination: {
+    type: String,
+    required: true,
+  },
+  revealedPreImage: {
+    type: String,
+    required: true,
+  },
+  roundedUpFee: {
+    type: Number,
+    required: true,
+  },
+  milliSatsFee: {
+    type: Number,
+    required: true,
+  },
+  hopPubkeys: [String],
+})
+
+const paymentAttemptSchema = Schema.Types.Mixed
+
+const paymentSchema = new Schema<LnPaymentType>({
+  createdAt: Date,
+  status: {
+    type: String,
+    enum: ["settled", "failed", "pending"],
```

Comment:
could be using a type instead? and like for `ledgerTransactionTypes` in ledger/schema.ts